### PR TITLE
Issue 2661: Fix Pravega concepts broken link

### LINKED ISSUE #2661: Incorrect link to concepts in the transactions documentation
**Problem description**
The concepts link is pointing to:

http://pravega.io/Pravega-Concepts/index.html

rather than:

http://pravega.io/docs/latest/pravega-concepts/

**Problem location**
Documentation

**Suggestions for an improvement**
Replace the link.

### PR DESCRIPTION
**Pull request by agarella**
**Change log description**
* Replaces the concepts link in the transactions documentation with the correct one.

**Purpose of the change**
Fixes #2661 

**How to verify it**
Click the link

### CODE REVIEW

**Comment by Prabha-Veerubhotla** [comment]
Hi, @agarella , the change looks good, but here are the guidelines you need to follow,  for this to go into the master. 
Please follow the below steps :
*  Create an issue with https://github.com/pravega/pravega/wiki/Contributing#creating-an-issue
*  Update the pull request as per the info here : 
 https://github.com/pravega/pravega/wiki/Contributing#creating-a-pull-request

File: documentation/src/docs/transactions.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Samples
 readme](https://github.com/pravega/pravega-samples/blob/master/standalone-examples/README.md).
 
 You really should be familiar with Pravega Concepts (see [Pravega
-Concepts](http://pravega.io/Pravega-Concepts/index.html)) before continuing reading this page.
+Concepts](https://github.com/pravega/pravega/blob/master/documentation/src/docs/pravega-concepts.md)) before continuing reading this page.
```
Comment (fpj) [inline documentation/src/docs/transactions.md]: Thanks for reporting and for the contribution, @agarella . I think the right link is this one, though:

http://pravega.io/docs/latest/pravega-concepts/

**Comment by fpj** [comment]
@agarella I have done most of the changes needed to get it to match our contribution guidelines to help you out as you are a first-time contributor. It is a small contribution, but important nonetheless, hopefully this is not the last. ;-)

**Comment by agarella** [comment]
Hey @fpj, thanks for making that easier! I've updated the link.

**Comment by agarella** [comment]
@fpj Ready to merge like this?

**Pull request merged**